library/_textfsm.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
```

Comment:
Same thing here. If this is for backwards compatibility, it's good. If it's the working code, then it should be renamed `library/_textfsm_parser.py`